RawVideo/RawVideo/ContentView.swift

```diff
@@ -0,0 +1,1032 @@
+            call = try await callAgent!.join(with: locator, joinCallOptions: joinCallOptions)
+            
+            try await call!.muteOutgoingAudio()
+            try await call!.muteIncomingAudio()
+            
+            let defaults = UserDefaults.standard
+            defaults.set(meetingLink, forKey: "MeetingLink")
+        }
+        catch let ex
+        {
+            callInProgress = false
+            
+            await ShowMessage(message: "Failed to start")
+            print(ex.localizedDescription)
+        }
+        
+        loading = false
+
+        if (call != nil)
+        {
+            call!.delegate = callObserver!
+            
+            AddRemoteParticipantList(remoteParticipantList: call!.remoteParticipants)
+        }
+    }
+    
+    func AddRemoteParticipantList(remoteParticipantList: [RemoteParticipant]) -> Void
+    {
+        remoteParticipantList.forEach { remoteParticipant in
+            remoteParticipant.delegate = remoteParticipantObserver
+            remoteParticipant.incomingVideoStreams.forEach { incomingVideoStream in
+                OnIncomingVideoStreamStateChanged(stream: incomingVideoStream)
+            }
+        }
+    }
+
+    private func CreateOutgoingVideoOptions() -> OutgoingVideoOptions
+    {
+        let rawOutgoingVideoStreamOptions = CreateRawOutgoingVideoStreamOptions()
+        switch (outgoingVideoStreamType)
+        {
+            case .localOutgoing:
+                let videoDeviceInfo = videoDeviceInfoList[selectedVideoDeviceInfoIndex]
+                localVideoStream = LocalVideoStream(camera: videoDeviceInfo)
+                localVideoStream!.delegate = localVideoStreamObserver
+                outgoingVideoStream = localVideoStream
+                
+                break
+            case .virtualOutgoing:
+                virtualOutgoingVideoStream = VirtualOutgoingVideoStream(videoStreamOptions: rawOutgoingVideoStreamOptions)
+                virtualOutgoingVideoStream!.delegate = virtualRawOutgoingVideoStreamObserver
+                outgoingVideoStream = virtualOutgoingVideoStream
+                
+                break
+            case .screenShareOutgoing:
+                screenShareOutgoingVideoStream = ScreenShareOutgoingVideoStream(videoStreamOptions: rawOutgoingVideoStreamOptions)
+                screenShareOutgoingVideoStream!.delegate = screenShareRawOutgoingVideoStreamObserver
+                outgoingVideoStream = screenShareOutgoingVideoStream
+                
+                break
+            default:
+                break
+        }
+
+        let outgoingVideoOptions = OutgoingVideoOptions()
+        var outgoingVideoStreamList = [OutgoingVideoStream]()
+        outgoingVideoStreamList.append(outgoingVideoStream!)
+        outgoingVideoOptions.streams = outgoingVideoStreamList
+        
+        return outgoingVideoOptions
+    }
+
+    private func CreateRawOutgoingVideoStreamOptions() -> RawOutgoingVideoStreamOptions
```

Comment:
Nit: Swift convention for this is `make` see section `Strive for Fluent Usage` in here https://www.swift.org/documentation/api-design-guidelines/
```suggestion
    private func makeRawOutgoingVideoStreamOptions() -> RawOutgoingVideoStreamOptions
```

Reply:
done